src/utils/fetchFunctions.js

```diff
@@ -26,6 +24,18 @@ function fetchArticleById(articleId){
 
 
 
+///api/articles/:article:id/comments
+function fetchCommentsByArticleId(articleId){
+
+    return axios.get(`https://nc-news-ldv7.onrender.com/api/articles/${articleId}/comments`).then((response) => {
+        return response.data.comments
+    })
+    .catch((err) => {
```

Comment:
Instead of having a catch block here, you could put it in the component and display a message to the user in case of an error.